tests/testsuite/package.rs

```diff
@@ -6143,7 +6156,7 @@ fn registry_not_inferred_because_of_multiple_options() {
             "Cargo.toml",
             r#"
             [workspace]
-            members = ["dep", "main", "other"]
+            members = ["dep", "main"]
```

Comment:
Should these test case changes be in a prior commit?

Reply:
Sorry, I think I messed up the rebase. It should be fixed now